Tests/RxBlockingTests/Observable+BlockingTest.swift

```diff
@@ -17,11 +17,11 @@ class ObservableBlockingTest : RxTest {
 
 extension ObservableBlockingTest {
     func testToArray_empty() {
-        XCTAssert(try! Observable<Int>.empty().toBlocking().toArray() == [])
+        XCTAssertEqual(try! Observable<Int>.empty().toBlocking().toArray(), [])
```

Comment:
The code was already was mixing both styles, so I guess one is better, but more importantly, you seem to get much better error messages when comparing objects rather than comparing the result of a boolean operation... so you get _[1,2,3] is not equal to [3,4,5]` with XCTAssetEqual rather than _assert failed_ or _false is not equal to true_ with XCTAssert.

I wasn't 100% sure of the exact difference in wording in Xcode, so I grabbed a screenshot.

<img width="459" alt="screen shot 2017-07-31 at 7 54 05 am" src="https://user-images.githubusercontent.com/323861/28757423-1c88082e-75c6-11e7-88d9-bdc2b674c9a9.png">